src/main/java/frc/robot/subsystems/turretSubsystem/TurretCommandFactory.java

```diff
@@ -376,7 +377,7 @@ public Trigger isReadyTrigger() {
   public boolean isReadyBoolean() {
     if (subsystem == null) return true;
     boolean ready =
-        subsystem.pitchAtSetpoint() && subsystem.turretAtSetPoint(Rotation2d.fromDegrees(2.5));
+        subsystem.pitchAtSetpoint(); // && subsystem.turretAtSetPoint(Rotation2d.fromDegrees(2.5));
```

Comment:
Check and see if this is necessary, and delete if we don't want to wait for the turret to be at a setpoint.  If we aren't rotating, we probably don't need it